airflow/migrations/versions/13f0be67a6da_runas.py

```diff
@@ -1,24 +0,0 @@
-"""runas
```

Comment:
This seems a left over from an earlier version. This should be removed from the commits entirely